packages/cli/src/constructs/project.ts

```diff
@@ -111,6 +111,13 @@ export class Project extends Construct {
           .filter((construct: Construct) => construct instanceof Check && construct.testOnly))
   }
 
+  getHeartbeatLogicalIds (): string[] {
+    return Object
+      .values(this.data.check)
+      .filter((construct: Construct) => construct instanceof Check && construct.constructor.name === 'HeartbeatCheck')
```

Comment:
We should have a HeartbeatCheck class as well, right? We can do a instance of directly on that